infra/import_graph.py

```diff
@@ -0,0 +1,94 @@
+import networkx as nx
```

Comment:
Suggest Google’s standard

```python
from __future__ import absolute_import
from __future__ import division
from __future__ import print_function
```

—it has really saved me from a world of confusion in the past.

Reply:
:+1: